components/datadog/fakeintake/ecsFargate.go

```diff
@@ -34,12 +34,12 @@ func NewECSFargateInstance(e aws.Environment) (*Instance, error) {
 	opts = append(opts, pulumi.Parent(instance))
 
 	alb, err := lb.NewApplicationLoadBalancer(e.Ctx, namer.ResourceName("lb"), &lb.ApplicationLoadBalancerArgs{
-		Name:           e.CommonNamer.DisplayName(pulumi.String("fakeintake")),
+		// Name:           e.CommonNamer.DisplayName(pulumi.String("fakeintake")),
```

Comment:
Is there a reason to comment this code and not to remove it?

Reply:
This was only a temporary quick workaround to unblock people, until a smarter fix (#240) is ready.
The goal of keeping the code commented was to not forget that, in the end, we would rather set an explicit name than letting pulumi generate one for us.
In #240, I’ve revisited all the locations where we use `(Name) DisplayName(…)`. If I had completely removed this piece of code, I would have missed those locations.